.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 120
```

Comment:
I've taken this value from the RuboCop config, and I'd like to set `100` (at both places), existing Ruby code corresponds to it, but it's up to you.

Reply:
Let's keep 120 chars per line (although I'm in 80-chars camp 😆)